src/lib.rs

```diff
@@ -1,8 +1,72 @@
 #![no_std]
 
-#[cfg(not(feature = "binary-vendor"))]
-mod contract;
+use app_io::*;
+use gmeta::Metadata;
+use gstd::{
+    errors::{ContractError, Result as GstdResult},
+    msg,
+    prelude::*,
+    ActorId, MessageId,
+};
+use hashbrown::HashMap;
+use hint::unreachable_unchecked;
 
-// See `Cargo.toml` for the description of the "binary-vendor" feature.
 #[cfg(feature = "binary-vendor")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
+
+static mut STATE: Option<HashMap<ActorId, u128>> = None;
+
+unsafe fn state_mut() -> &'static mut HashMap<ActorId, u128> {
```

Comment:
```suggestion
fn state_mut() -> &'static mut HashMap<ActorId, u128> {
```

Reply:
I'd like to leave it unsafe since `.unchecked_unwrap()` can lead to UB if somebody will use this before a state initialization.